assignment/assignment-impl/impl/src/java/org/sakaiproject/assignment/impl/BaseAssignmentService.java

```diff
@@ -10333,6 +10331,21 @@ private int getReviewScore(ContentResource cr)
 			}
 		}
 		
+		public String getContentReviewSubmitterId(ContentResource cr){
+			//Group submissions store the group ID as the submitterId, so find an actual user ID
```

Comment:
Would be nice to have this as the JavaDoc on the method rather than just as a comment inside.